duckduckgo/Android
# add additional language files

Task/Issue URL: https://app.asana.com/0/715106103902962/1134060828640424/f
Tech Design URL: 
CC: 

**Description**:
Adds translations for 21 extra languages

**Steps to test this PR**:
1. Test the app in English, and ensure there are no changes
1. Set your device language to some of the languages included in this PR, and check the translations look to be valid (as best as you can, given you might not understand the language!)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
